core/sr-primitives/src/generic/unchecked_extrinsic.rs

```diff
@@ -85,7 +85,7 @@ where
 	AccountId: Member + MaybeDisplay,
 	Context: Lookup<Source=Address, Target=AccountId>,
 {
-	type Checked = CheckedExtrinsic<AccountId, Index, Call>;
+	type Checked = CheckedExtrinsic<AccountId, Index, Call, u32>;
```

Comment:
So `()` is indeed nicer but won't work because it doesn't fulfill the trait bounds and the boilerplate needed for it is too much afaict. type alias-ed.